helm-org.el

```diff
@@ -58,9 +58,11 @@ NOTE: This will be slow on large org buffers."
   (org-show-entry))
 
 (cl-defun helm-source-org-headings-for-files (filenames
-                                              &optional (min-depth 1) (max-depth 8))
+                                              &optional (min-depth 1) (max-depth 8) (parents nil))
   (helm-build-sync-source "Org Headings"
-    :candidates (helm-org-get-candidates filenames min-depth max-depth)
+    :candidates (helm-org-get-candidates filenames min-depth max-depth parents)
+    :candidate-transformer (if parents 'reverse nil)
+    :after-init-hook 'helm-follow-mode
```

Comment:
I've tried several ways, but I can't find the right way to enable `helm-follow-mode`.  It needs to be enabled *after* the Helm buffer is created, because if it's already enabled when the buffer is created, it "follows" to the first candidate immediately, before the `:preselect` has a chance to select the current heading, which causes the point to jump to the top of the buffer and back down again.  It's not big deal, but it's a bit distracting, and there must be a way around it.  :)